### ISSUE TITLE
Theme does not get ported to the React Dev Tools

### ISSUE THREAD

**Issue by jamby** [micjamking/devtools-author#144]
So when using React, the React Dev Tools will appear in the dev tools area. But when I go to it, I lose the style that would appear on the Inspect section.

For instance:

![screen shot 2016-06-09 at 4 48 35 pm](https://cloud.githubusercontent.com/assets/250728/15945800/121321c8-2e62-11e6-884f-485ff1cec324.png)
![screen shot 2016-06-09 at 4 48 44 pm](https://cloud.githubusercontent.com/assets/250728/15945801/152efd8c-2e62-11e6-9baf-6a95becb5cbd.png)

**Comment by rorysmorris** [comment]
Even Chrome's built-in dark theme doesn't support React Dev Tools. There's an issue for it on React Dev Tools's Github page, it didn't sound like a high priority. 3am eye-burn is us!

**Comment by jamby** [comment]
Yeah I noticed one guy on Twitter that's working on it! Which is great.

**Comment by rorysmorris** [comment]
Ah - awesome, eyes peeled!
